debug/bright-v7.json

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
@jfirebaugh Ah yeah sorry, that was sloppy -- it's actually a copy of `bright-v7.json` with `interactive: true` set on the street styles.  Will fix